README.md

```diff
@@ -103,41 +102,50 @@ need to point to `localhost` (or to your virtual machine, if that's
 how you develop things). For instance, here's an `/etc/hosts` entry
 using `localhost` (put this in addition to the `localhost` line
 already in there):
-  
-    127.0.0.1		apollo13.dev.spacelog.org mercury6.dev.spacelog.org dev.spacelog.org
 
-and here's one for a virtual machine (you'll need to change the dotted
-quad at the start of the line):
-  
-    192.168.56.101	apollo13.dev.spacelog.org mercury6.dev.spacelog.org dev.spacelog.org
+    127.0.0.1		apollo13.dev.spacelog.org mercury6.dev.spacelog.org dev.spacelog.org
 
 ### Reindexing
 
 Whenever you edit information about a mission, or add a new one, you
-need to run `make reindex` again. If you get errors you may find the
-`lognag.pl` script in `mcshred/src` useful: just give it some
-transcript files and it'll tell you where it finds possible errors or
-weirdnesses. (For new missions, you'll probably have to add things
-into the valid speakers list at line 71.)
-
-Note that a full `make reindex` can take a while, so you can index
-just a single mission by doing `ENV/bin/python -m backend.indexer ma6`
-or similar (or just `python -m backend.indexer ma6` if you aren't
-using a virtualenv.
+need to reindex the content again to see it in your local environment.
+
+If you're running in Docker (as we recommend), reindexing happens as
+part of the container build process. You can manually rebuild the
+container with `docker compose build` or you can have it happen
+automatically as needed by rebuilding the container every time you
+start your environment using `docker compose up --build` (which is a
+little slower than starting a container normally, but might be easier
+if you're working on missions content).
+
+If you're running without Docker (or working inside the container
+manually), you can reindex using `make reindex`.
+
+If you get errors you may find the `lognag.pl` script in `mcshred/src`
+useful: just give it some transcript files and it'll tell you where it
+finds possible errors or weirdnesses. (For new missions, you'll
+probably have to add things into the valid speakers list at line 71.)
+
+Note that a full reindex can take a while (usually about 5 minutes), so
+you can index just a single mission by running `python3 -m
+backend.indexer mission_id` (where `mission_id` is the name of the
```

Comment:
Maybe make the metavar `MISSION_ID` so it's visually distinct from the rest of the command?